src/preload-safe-debug.ts

```diff
@@ -16,14 +16,16 @@
  * must be a preload script.
  */
 module.exports = function debug(namespace: string): (...params: any[]) => void {
+  // if debugging is not requested, just return
```

Comment:
@kris7t - could you please confirm if this change is correct and it doesn't affect the logic of why this preload script was created in the first place?
Unfortunately, without this change, the `npm test` command fails with an error.

Reply:
I added an alternative fix to the branch instead.

Reply:
tx @kris7t . i have cherry-picked your commit into the `develop` branch so that its used/useful outside of this PR.